meetings/2025-01-21.md

```diff
@@ -0,0 +1,26 @@
+# Meeting on 2025-01-21
+
+Present: Cornelius, Florian, Frederik, Loic, Max, Peter, Tristram
+
+## Agenda
+
+* Chair of Technical Committee
+  * Agree on details of voting procedure (see https://github.com/OpenRailAssociation/technical-committee/blob/main/charter.md#chair-of-the-tc)
+  * Candidates
+* Criteria for incubation stage 2 (https://github.com/OpenRailAssociation/technical-committee/milestone/5)
+* Annual report
+* Nextcloud
+  
+## Protocol
+
+* Criteria for incubation stage 2 (https://github.com/OpenRailAssociation/technical-committee/milestone/5)
+  * Open communication: we cover it as an additional explanation on the open governance criteria
+  * There is one small thing left regarding documenting open communication (see issues). Other than that the criteria are done.
+  * OSRD questionnaire needs to be rebased. Next week we will have a look and discuss it.
+* Chair of Technical Committee
+  * Each TC member has a vote. We have 8 members, 5 project representatives and 3 appointed members by the board. At least 5 votes is a majority.
+  * The 3 appointed members are Cornelius, Peter and Loic. There is no end date for the appointment. It could also be less than 3 people. Do we want to continue with the current set of appointed people or would we want to recommend to the board to change the setup? We have a consensus that the current setup is good for now. We will discuss the question if and what to change there later. The appointed members are part of the TC and can run for the chair and vote for the chair.
```

Comment:
```suggestion
  * The 3 appointed members are Cornelius, Peter and Loïc. There is no end date for the appointment. It could also be less than 3 people. Do we want to continue with the current set of appointed people or would we want to recommend to the board to change the setup? We have a consensus that the current setup is good for now. We will discuss the question if and what to change there later. The appointed members are part of the TC and can run for the chair and vote for the chair.
```